docs/azure/services/Azure_Functions.md

```diff
@@ -0,0 +1,109 @@
+
+#### Inbound Access Restrictions
+This option, which is available for all hosting plans, allows users to define a list of IP addresses which are allowed or denied access to a function. When no entries have been defined, an implicit “deny all” is applied.
+
+The following command will output a Function’s inbound access restrictions:
+```CLI
+az functionapp config access-restriction show -g ResourceGroup -n AppName
+```
+#### Outbound Traffic
+##### VNet Integration
+One of the ways to control outbound network access on a function is to set up a Virtual Network integration. This provides a function with the ability to communicate to resources within a dedicated VNet’s subnet. 
+
+This configuration can be found under the `Networking` blade or by running the following command:
+```CLI
+az functionapp vnet-integration list --name --resource-group
+```
+
+Once the VNet integration has been configured, the function will not by default have access to Azure resources. This would require a specific configuration following the change to ensure communication is possible with other internal resources.
+
+There are two VNet integration options available:
+- Regional VNet integration - when the function is connecting to resources in the same region.
+- Gateway-required VNet integration – when the function is connecting to VNets in different regions.
+
+When reviewing Gateway-required VNet integrations, the following security features are important to keep in mind:
+- If a VNet has got peering connections set up with other VNets or VPNs, the connection from the function App will be transitive, causing resources in additional networks to be reachable.
+- Access to resources across Azure ExpressRoute or service endpoints is not enabled.
+
+When reviewing a regional VNet integration, the Function App will be able to access:
+- Resources within the same VNet
+- Resources in VNets peered to the VNet the app is integrated with
+- Service endpoint secured services
+- Resources across Azure ExpressRoute connections
+- Resources across peered connections which include ExpressRoute connections
+- Private endpoints
+
+Network Security Groups (NSGs) and Route Tables (UDRs) can be applied against dedicated subnets of integrated VNets. Standard security practices following the principle of least privilege should be followed in that situation. 
+
+The assessment notes on best security practices for NSGs can be found [here](https://learn.microsoft.com/en-us/azure/virtual-network/network-security-groups-overview) and for UDRs [here]( https://learn.microsoft.com/en-us/azure/virtual-network/vnet-integration-for-azure-services). However, it is worth noting that inbound rules will not affect the inbound access to the function, which is controlled by the access restrictions at the service-level configuration that were previous described.
```

Comment:
I wouldn't really say that the two links represent best security practices as they are mainly the baseline documentation for the service feature. It would be better if we phrased it instead and say what would be some high-level recommendations for the two features when dealing with functions.